### PR TITLE
Clarified Taylor importance factors theory documentation

### PR DESCRIPTION
**Pull request by mbaudin47**

### CODE REVIEW

**Comment by jschueller** [comment]
the doc fails to build

**Comment by mbaudin47** [comment]
The doc now builds. This PR seems good to go to me.

File: python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst [hunk 1]
```diff
@@ -3,71 +3,82 @@
...
+   evaluated at the point :math:`\ux`.
 
+Therefore the expectation of :math:`Z` is: 
 
   .. math::
 
-     \underline{\cF} = \nabla h(\muX) \times \frac{\Cov \uX .^t\nabla h(\muX)}{\Var Z}
+     \Expect{Z} = h(\muX).
 
+The independence of the marginals imply:
```
Comment (jschueller) [inline python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst]: implies

**Comment by jschueller** [comment]
see #1079

**Pull request closed without merging**